script/session.php

```diff
@@ -66,6 +66,9 @@
   if ($_SERVER["SERVER_ADDR"] == "10.144.245.30") {
     $use_sts = false;
   }
+  if (preg_match('/^PHP (\d+\.)?(\d+\.)?(\*|\d+) Development Server$/', $_SERVER["SERVER_SOFTWARE"]) === 1) {
```

Comment:
Why was this added? Can you revert this change?

Reply:
They said why
> so one can use PHP's built-in webserver on a local development environment. This can be safely ignored if it doesn't fit.

Reply:
```suggestion

```

Reply:
I completely missed that. I would prefer this to be removed.

Reply:
If you don't use this change, the whitespace changes in the file should also be reverted, so the commit needn't touch it.

Reply:
@dlemstra sure! Just cherry-pick the desired changes and ignore the rest.

Or do you want me to create a new PR without the session code?

Reply:
Merged it and removed that part.

Reply:
No worries. It won't affect the updated OpenCL content in any way. Thank you so much for merging it, @dlemstra.

Reply:
> Merged it and removed that part.

@dlemstra the session.php changes still landed in Website@689b0fc?

Reply:
They did but I pushed a patch to remove it: 0c4751d3aea763b84c599b587a605889a1490c70